lib/Network.jsx

```diff
@@ -1,6 +1,5 @@
 import $ from 'jquery';
```

Comment:
should we be removing/replacing this too?

Reply:
We decided to not migrate it because we don't have access to repos that are actually using this `Network` file, and we couldn't test it. Replacing jQuery can introduce many critical regressions and require bigger logic changes